### ISSUE TITLE
Search box missing label for screen readers

### ISSUE THREAD

**Issue by aamct2** [storybooks/storybook#6130]
**Describe the bug**

The search box in the sidebar does not have a `<label>` or `aria-label` attribute set to be used by screen readers (only one is necessary).

This is important to meet WCAG 2.1 (Level A) [1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) and [3.3.2](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html). Deque University also provides a [good description](https://dequeuniversity.com/rules/axe/3.1/label).

**To Reproduce**
1. Load Storybook
1. Inspect the search box
1. No `<label>` or `aria-label` attribute is set

Alternatively, run an automated accessibility checker (such as Lighthouse in the Chrome Developer Tools) to see it picked up.

**Expected behavior**

All inputs should have a `<label>` or `aria-label` attribute (or similar) available for screen readers.

**System:**
 - OS: macOS 10.14.3
 - Browser: Chrome 73.0.3683.75
 - Version: `next` branch as of Commit 638cbd5fcb383377f52b295607f6a8b0e95a6c7d

**Comment by shilman** [comment]
Yo-ho-ho!! I just released https://github.com/storybooks/storybook/releases/tag/v5.1.0-alpha.8 containing PR #6131 that references this issue. Upgrade today to try it out!

Because it's a pre-release you can find it on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
